OCPBUGS-30076-av (#74785)
openshift/openshift-docs: modules/creating-config-files-cluster-install-oci.adoc

<1> The cluster name that you specified in your DNS record.
<2> The namespace of your cluster on {product-title}.
<3> If you use IPv4 as the network IP address format, ensure that you set the `rendezvousIP` parameter to an IPv4 address that the VCN’s Classless Inter-Domain Routing (CIDR) method allocates on your network. Also ensure that at least one instance from the pool of instances that you booted with the ISO matches the IP address value you set for `rendezvousIP`.
<4> The URL of the server where you want to upload the rootfs image.

. Apply one of the following two updates to your `agent-config.yaml` configuration file:
+
* For a disconnected network:  After you run the command to generate a minimal ISO Image, the Agent-based installer saves the rootfs image into the `./<installation_directory>/boot-artifacts` directory on your local system. Use your preferred web server, such as any Hypertext Transfer Protocol daemon (`httpd`), to upload rootfs to the location stated in the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file.
+

> For example, if the `bootArtifactsBaseURL` parameter states \http://192.168.122.20, you would upload the generated rootfs image to this location, so that the Agent-based installer can access the image from \http://192.168.122.20/agent.x86_64-rootfs.img. After the Agent-based installer boots the minimal ISO for the external platform, the Agent-based Installer downloads the rootfs image from the \http://192.168.122.20/agent.x86_64-rootfs.img location into the system memory.

Reviewer: ```suggestion
For example, if the `bootArtifactsBaseURL` parameter states `\http://192.168.122.20`, you would upload the generated `rootfs` image to this location, so that the Agent-based installer can access the image from `\http://192.168.122.20/agent.x86_64-rootfs.img`. After the Agent-based installer boots the minimal ISO for the external platform, the Agent-based Installer downloads the `rootfs` image from the `\http://192.168.122.20/agent.x86_64-rootfs.img` location into the system memory.
```

Reviewer: I think you may need some backticks here?

Author: AH, OK. I'll try this agin and hopefully the backslash does not render.
![Screenshot from 2024-04-18 09-24-59](https://github.com/openshift/openshift-docs/assets/57954076/52670f71-507c-4598-97ce-bfa553b2f374)

+
[NOTE]
====

> The Agent-based Installer also adds the value of the `bootArtifactsBaseURL` to the minimal ISO Image’s configuration, so that when the Operator boots a cluster’s node, the Agent-based Installer  downloads the rootfs image into system memory.

Author: :heavy_check_mark:

====
+

> * For a connected network: You do not need to specify the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file, because the Agent-based Installer, by default, reads the a rootfs URL location from \https://rhcos.mirror.openshift.com. After the Agent-based Installer  boots the minimal ISO for the external platform, the Agent-based Installer then downloads the rootfs file into your system’s memory from the default {op-system} URL.

Reviewer: ```suggestion
* For a connected network: You do not need to specify the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file. The default behavior of the Agent-based Installer is to read the rootfs URL location from \https://rhcos.mirror.openshift.com. After the Agent-based Installer boots the minimal ISO for the external platform, the Agent-based Installer then downloads the rootfs file into your system’s memory from the default {op-system} URL.
```

Reviewer: Another extra space and a minimalism suggestion to break up a complex sentence.

Author: :heavy_check_mark:

+
[IMPORTANT]
====

> Consider that the full ISO image, which is in excess of `1` GB, includes the rootfs image and the image is considerably larger than the minimal ISO Image, which is typical less than `150` MB.

Reviewer: ```suggestion
Consider that the full ISO image, which is in excess of `1` GB, includes the rootfs image. The image is larger than the minimal ISO Image, which is typically less than `150` MB.
```

Reviewer: Minimalism suggestion

Author: :heavy_check_mark:
